runtime/src/main/java/org/corfudb/runtime/view/AbstractView.java

```diff
@@ -110,8 +111,11 @@ public Layout getCurrentLayout() {
                     log.warn("Server still not ready. Waiting for server to start "
                             + "accepting requests.");
                     Sleep.sleepUninterruptibly(retryRate);
-                } else if (re instanceof WrongEpochException) {
-                    WrongEpochException we = (WrongEpochException) re;
+                } else if (re instanceof WrongEpochException
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace the usage of the "instanceof" operator by a catch block. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1193)